### ISSUE TITLE
auto-query/retrieve does not work anymore with 2.0

### ISSUE THREAD

**Issue by amareschal** [horosproject/horos#104]
Hi Guys, 

I am running the new 2.0 version of Horos and El Capitan on an imac 5k (late 2014). 
I have to manually retrieve images because they don't come automatically anymore. 
If I press query from the auto-query/retrieve window, there are no cases showing up on the list, whereas if I manually query them from the database window, I can see the list of cases and retrieve the cases from there.  Is there a different box that need to be checked for auto-query retrieve on the new version?
I have attached a screen picture of the auto-query/retrieve window after pressing query. 

Thank you very much in advance for any help you may be able to provide.

Sincerely.

Augustin

<img width="1168" alt="screen shot 2016-06-08 at 10 43 41 am" src="https://cloud.githubusercontent.com/assets/16354647/15904832/56b44a10-2d67-11e6-840d-0f349bae8c52.png">

**Comment by amareschal** [comment]
Hi Guys, 

Me again, I also upgraded one of the computer in our hospital (imac 2011 running el capitan) with 2.0 and got the same problem. This seems to narrow down our problem to Horos or the way our network is set up.

Thank you again

Augustin

**Comment by Jaswant-Singh** [comment]
I am a newbie - please help!!
I upgraded to Horos 2 today. It did another import from my Osirix (copied files) but no patients appeared in the database window. My previous Horos database has also disappeared. When I import any new Dicom folder into the database, it seem to copy the files to data directory (inside my database location), but no patients appears in the database. Thanks

**Comment by amareschal** [comment]
I am sorry to heart that, 
Mine did not disappear and I can still import case from the PACS as long as I query retrieve them manually.

**Comment by fvpolpeta** [comment]
Hi,

We have been working to fix the Auto Query&Retrieve Bug. 

In respect to Database issues, from 1.1.7 to 2.0.0, nothing was changed in the database model. You can rollback to 1.1.7 if you wish.

But you can try firstly to start Horos in Safe Mode (Command + Option + Click the App) and rebuild the database (File => Rebuild Entire Database)

**Comment by Jaswant-Singh** [comment]
Thanks a lot for getting back to me. Kudos for keeping up the wonderful project and the support!
I tried re-building database - it finds and scans some 12500 files (size of my “Horos data” folder is 3.6Gb) but nothing shows up in the albums or main window of database. If I import and link a new directory, I can see the content but if I try to import and copy files, it does not show in the database but seems to copies the files (in the activity window).
I am using custom-location for my database that is located on a 128Gb SD card attached to my Macbook pro 15” (late 2013 model). Thanks a lot for your help again.

Regards
Jaswant Singh

---

**Comment by fvpolpeta** [comment]
You are welcome.

Please, keeping a backup of your files in a separate and safe folder, try to clean up all your installation:

With Horos App NOT running remove the following files/dirs:

~/Documents/Horos Data     (THIS IS WHERE DICOM FILES AND HOROS DB ARE ORIGINALLY STORED. PLEASE BACKUP IF NECESSARY)

~/Library/Preferences/com.horosproject.horos.plist
~/Library/Cookies/com.horosproject.horos.binarycookies
~/Library/Caches/com.horosproject.horos
~/Library/Application Support/Horos

( ~ = User home directory)

**Comment by fvpolpeta** [comment]
Hey there. There was a run-time exception not handled during QueryController initialization. This can be the reason why the AutoQueryRetrieve stopped working. Luckily it was working before.

Please try the following build and let me know your results:

https://dl.dropboxusercontent.com/u/15111870/PurView/HorosNightlyRC4_20160612.dmg

**Comment by fvpolpeta** [comment]
New build: https://dl.dropboxusercontent.com/u/15111870/PurView/HorosNightlyRC4_20160613.dmg

Please, let us know if the issue with Q&R was addressed.

**Comment by amareschal** [comment]
Thank you Fauze,

I'll be in tomorrow morning, I'll try it first and will let you know.
Thanks again.

**Comment by fvpolpeta** [comment]
Confirmed by the community, the following build solves tge issue.

https://dl.dropboxusercontent.com/u/15111870/PurView/HorosNightlyRC4_20160613.dmg

**Comment by amareschal** [comment]
Tried your new version but it did not work. I attached a screen shot that
shows 5 attempts to autoquery retrieve (at the bottom left of the picture).
The cases that are already retrieved in the DICOM auto Query/Retrieve
window in green were retrieved the same day with the 1.1.7 version of Horos
before I upgraded.
Though, there is some improvement because, before with the 2.0, the cases
were not visible at all in the auto-query retrieve window and there was
nothing showing "auto-query images..."  in the activity section of the
database.

Thank you.

Augustin
